plugin/teksi_wastewater/metadata.txt

```diff
@@ -1,19 +1,19 @@
 
 [general]
 name=TEKSI Wastewater
-description=Plugin to manage wastewater networks (formerly: QGEP) # // skip-keyword-check
-qgisMinimumVersion=3.2
+description=TEKSI Wastewater plugin to manage wastewater networks. Provide selection, connection and editing tools.
```

Comment:
In qgep we had the following text: The QGEP plugin adds functionality to manage wastewater networks in QGIS. It requires a QGEP compatible database and project file.

Do we want to add the second part again?
Do we want to add something as - It supports Swiss standards SIA405 wastewater, VSA-DSS and VSA-KEK  (2020.x) for INTERLIS import / export. Or would this something to be added on the homepage?